bait-files/src/main/webapp/register.css

```diff
@@ -0,0 +1,43 @@
+body {
+    font-family: 'Roboto', sans-serif;
+    padding: 100px;
+    text-align: center;
+}
+
+form {
+  border: 1px solid #CCC;
+  border-radius: 1em;  
+  margin: 0 auto;
+  padding: 2em;
+  width: 500px;
+}
+
+ul {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+form li + li {
+  margin: 3em;
+}
+
+input {
+  border: 1px solid #999;
+  width: 300px;
+  text-align: center;
+}
+
+input:focus, textarea:focus {
+  border-color: #000;
+}
+
+button {
+    background: #0066A2;
```

Comment:
Indentation (two spaces) and selector alphabetization.

Reply:
Resolved!